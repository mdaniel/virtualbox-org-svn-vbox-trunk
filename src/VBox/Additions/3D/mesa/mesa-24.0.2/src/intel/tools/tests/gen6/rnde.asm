rnde(8)         g6<1>F          g3<8,8,1>F                      { align1 1Q };
rnde(16)        g8<1>F          g4<8,8,1>F                      { align1 1H };
