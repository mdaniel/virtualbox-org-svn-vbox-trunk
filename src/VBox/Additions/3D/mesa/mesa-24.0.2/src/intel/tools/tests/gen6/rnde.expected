46 00 60 00 bd 03 c0 20 60 00 8d 00 00 00 00 00
46 00 80 00 bd 03 00 21 80 00 8d 00 00 00 00 00
